common/persistence/nosql/nosqlplugin/cassandra/db.go

```diff
@@ -72,3 +73,29 @@ func (db *cdb) IsThrottlingError(err error) bool {
 func (db *cdb) IsDBUnavailableError(err error) bool {
 	return db.client.IsDBUnavailableError(err)
 }
+
+func (db *cdb) isCassandraConsistencyError(err error) bool {
+	return db.client.IsCassandraConsistencyError(err)
+}
+
+func (db *cdb) executeDeleteQuery(q gocql.Query) error {
```

Comment:
I suggest naming those more generally. There is nothing related to deletes in those helpers. Perhaps `executeWithConsistencyAll` / `executeBatchWithConsistencyAll` ?